cpp/velox/shuffle/VeloxSplitter.h

```diff
@@ -0,0 +1,370 @@
+#pragma once
+
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "velox/type/Type.h"
+#include "velox/vector/ComplexVector.h"
+#include "velox/vector/FlatVector.h"
+
+#include <arrow/filesystem/filesystem.h>
+#include <arrow/filesystem/localfs.h>
+#include <arrow/io/api.h>
+#include <arrow/ipc/writer.h>
```

Comment:
Will Arrow IPC writer be used in velox splitter? or probably the headers need to be minimized

Reply:
OK, I will check it, and we should minimize the header files to include

Reply:
yes, structs such as `arrow::ipc::IpcPayload` and `arrow::ipc::IpcWriteOptions` are used, so including these header files is necessary.